src/rules/Default/ScrapDefault.ts

```diff
@@ -33,7 +38,7 @@ export default async (url, htmlDoc) => {
       getAttrOfDocElement(htmlDoc, 'meta[name="twitter:image:src"]', 'content'),
       getAttrOfDocElement(htmlDoc, 'meta[name="twitter:image"]', 'content'),
       getAttrOfDocElement(htmlDoc, 'meta[itemprop="image"]', 'content'),
-    ].filter(i => !isEmpty(i)),
+    ].filter(i => !isEmpty(i)).map((i => fixRelativeUrls(baseUrl, i))),
```

Comment:
Nice normalization to urls, but can we have `fixRelativeUrls` within filter scope like this
`
filter(i => {
   if(!isEmpty(i)){
      return fixRelativeUrls(baseUrl, i)
   } 
})`
 rather than `filter` -> `map`

Reply:
This will disrupt readability while webpack should transpire to the same performance.

Reply:
Agree to have map